CONTRIBUTING.md

```diff
@@ -79,10 +79,52 @@ npm run lib
 
 #### Launch a development server with a real Kedro project
 
-Run the following command:
+Before launching a development server with a real Kedro project, you'd need to have [Python](https://www.python.org/)(>=3.6, <3.9) installed. 
+
+> *Optional*: We strongly recommend setting up [conda](https://docs.conda.io/en/latest/) to manage your Python versions and virtual environments.
+
+After setting up Python, install latest Kedro with:
+
+```bash
+pip3 install kedro
+```
+
+Create a real Kedro project with the `spaceflights` example:
```

Comment:
Thanks for adding the extensive details Lim! My only concern with specifically adding in the need to use the `spaceflights` project will cause confusion for anyone coming in to think that this only works with this project, which deviates with the original intention that the contributor can run the dev server with any kedro project they have. It might be worth to further mention that this step is only needed if they are starting Kedro from scratch?

Reply:
I added a clarifying note.